OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -547,14 +547,9 @@ void ITick.Tick(Actor self)
 				b.Tick();
 
 			using (new PerfSample("tick_bots"))
-			{
-				Sync.CheckSyncUnchanged(World, () =>
-				{
-					foreach (var t in tickModules)
-						if (t.IsTraitEnabled())
-							t.BotTick(this);
-				});
-			}
+				foreach (var t in tickModules)
```

Comment:
I would prefer that you add config for this. Bots have unsynced behavior inside what the rest of the system thinks is synced context.